src/cmdline/console.rs

```diff
@@ -0,0 +1,347 @@
+// Copyright 2022 Red Hat, Inc.
```

Comment:
This module is both terrifying and great, which is about what we expected I think. :laughing:

Really nice work!